fitters/include/DAF.h

```diff
@@ -55,6 +55,12 @@ class DAF : public AbsKalmanFitter {
 
  public:
 
+  /**
+   * @brief Create DAF. Per default, use KalmanFitterRefTrack as fitter.
```

Comment:
The documentation here is identical to the constructor below. Please write down what's the difference between the two constructors and what's the advantage of using this w.r.t. the other.

Reply:
done